drivers/ios-driver/src/nativeTest/kotlin/com/squareup/sqldelight/drivers/ios/NativeSqliteDriverTest.kt

```diff
@@ -47,8 +47,7 @@ class NativeSqliteDriverTestMemory : NativeSqliteDriverTest() {
 
 abstract class NativeSqliteDriverTest : LazyDriverBaseTest() {
 
-  //Can't run this till https://github.com/touchlab/SQLiter/issues/8
-  /*@Test
+  @Test
   fun `close with open transaction fails`(){
```

Comment:
this test uses the old `getConnection()` api which doesnt exist anymore

Reply:
Ah, yeah. Forgot ios tests need a different task run. Updating.

Reply:
yea sgtm